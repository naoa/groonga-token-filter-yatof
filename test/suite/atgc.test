register token_filters/yatof

tokenize TokenBigram \
  "ATGCATGCA ATKING" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterATGC

