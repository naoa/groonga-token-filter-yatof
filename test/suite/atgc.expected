register token_filters/yatof
[[0,0.0,0.0],true]
tokenize TokenBigram   "ATGCATGCA ATKING"   --normalizer NormalizerAuto   --token_filters TokenFilterATGC
[[0,0.0,0.0],[{"value":"atking","position":0,"force_prefix":false}]]
